src/App/Game/GameBoardWrapper.test.tsx

```diff
@@ -13,26 +13,41 @@ class MockBoard extends React.Component<any, any> {
   }
 }
 
-test('show no warning', () => {
-  const Board = gameBoardWrapper({
-    board: MockBoard,
-    gameArgs: {
-      gameCode: 'chess',
-      mode: GameMode.OnlineFriend,
-    },
+describe('GameBoardWrapper', () => {
+  it('should not show warning', () => {
+    const Board = gameBoardWrapper({
+      board: MockBoard,
+      gameArgs: {
+        gameCode: 'chess',
+        mode: GameMode.OnlineFriend,
+      },
+    });
+    const wrapper = Enzyme.mount(<Board isConnected={true} G={{ pgn: '' }} ctx={{}} />);
+    expect(wrapper.find('AlertLayer').length).to.equal(0);
+  });
+
+  it('should show disconnected warning', () => {
+    const Board = gameBoardWrapper({
+      board: MockBoard,
+      gameArgs: {
+        gameCode: 'chess',
+        mode: GameMode.OnlineFriend,
+      },
+    });
+    const wrapper = Enzyme.mount(<Board isConnected={false} G={{ pgn: '' }} ctx={{}} />);
+    expect(wrapper.html()).to.contain('Connection lost');
   });
-  const el = Enzyme.mount(<Board isConnected={true} G={{ pgn: '' }} ctx={{}} />);
-  expect(el.find('AlertLayer').length).to.equal(0);
-});
 
-test('show disconnected warning', () => {
-  const Board = gameBoardWrapper({
-    board: MockBoard,
-    gameArgs: {
-      gameCode: 'chess',
-      mode: GameMode.OnlineFriend,
-    },
+  it('should dismiss sharing', () => {
+    const Board = gameBoardWrapper({
+      board: MockBoard,
+      gameArgs: {
+        gameCode: 'chess',
+        mode: GameMode.OnlineFriend,
+      },
+    });
+    const wrapper = Enzyme.mount(<Board isConnected={true} G={{ pgn: '' }} ctx={{}} />);
+    (wrapper.instance() as any)._dismissSharing();
```

Comment:
what does this do? We dont have this functionality in the UI. Maybe we should delete the implementation instead?

Reply:
You are right.  We used _dismissSharing before lobby so I didn't realize.  Removed the test and implementation.